tests/unit/App.spec.ts

```diff
@@ -349,7 +348,7 @@ describe.skip('Numbered company setup', () => {
     localVue.use(VueRouter)
     const router = mockRouter.mock()
     await router.push({ name: 'define-company', query: { id: 'T7654321' } })
-    // Still need store for sbc-common-components
```

Comment:
Nice comment but we're not actually injecting "store" when mounting so I deleted the comment.

Reply:
As long as no one uninstalls Vuex completely..  I remember doing that for FAS and it sbc-common-components stopped working.

Reply:
I see you left a comment in [main.ts](https://github.com/bcgov/business-edit-ui/blob/c6d6c5c4728d2d8b5a388c6e0198870de25584c5/src/main.ts#L107), which is the only place where the Vuex store is injected. I think that's sufficient. Do you agree?